lib_neutral_prompt/cfg_denoiser_hijack.py

```diff
@@ -55,7 +72,9 @@ def get_webui_denoised(
 
 
 def get_cfg_rescale_factor(cfg_cond, cond):
-    return global_state.cfg_rescale * (torch.std(cond) / torch.std(cfg_cond) - 1) + 1
+    cfg_rescale_factor = global_state.cfg_rescale * (torch.std(cond).item() / torch.std(cfg_cond).item() - 1) + 1
+    CFGRescaleFactorSingleton.set_cfg_rescale_factor(cfg_rescale_factor)
```

Comment:
this isn't right: you want to override `global_state.cfg_rescale` above, as `cfg_rescale_factor` is dynamic and varies with the std of the latents at each step.